src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -271,12 +271,17 @@ ServerHandlerImp::onWSMessage(
                 sb.commit(boost::asio::buffer_copy(
                     sb.prepare(n), boost::asio::buffer(p, n)));
             });
+        JLOG(m_journal.trace())
+            << "Websocket sending '" << jvResult << "'";
```

Comment:
I believe Json::stream may be a co-conspirator here. I've not investigated deeply, but after logging the json tokens, and noticing progress, albeit extremely slowly, I suspect we must be doing a lot of needless reallocation/copying somehow. 

replacing Json::stream with
```c++
            auto const jrs = to_string(jr);
            auto n = jrs.length();
            void * p = (void *) jrs.c_str();
            sb.commit(boost::asio::buffer_copy(
                        sb.prepare(n), boost::asio::buffer(p, n)));
```

And I'm avoiding the stalls.

Next I tried preallocating the streambuf:
```c++
            auto const jrs = to_string(jr);
            beast::streambuf sb (jrs.length());
```

So ? We need some kind of heuristic to get a size hint ?  Or ?

In any case, later on we have code that looks like this:
```
            session->send(std::make_shared<
                StreambufWSMsg<decltype(sb)>>(std::move(sb)));
```

With the growth characteristics observed from afar (and the simple allocation hint tweak), this doesn't seem any better than just serialising to a string? How is this all supposed to work?